tests/Base.nut

```diff
@@ -0,0 +1,81 @@
+// MIT License
```

Comment:
Is there a reason why not to have a base class that extends ImpTestCase, which would be used for all the test classes to extend on their turn?